app/Svg/Parser.hs

```diff
@@ -159,8 +160,8 @@ parseTextHelper key styles' trans textTags =
         currTrans = getTransform $ head textTags
         newTrans = addTuples trans currTrans
         alignAttr = styleVal "text-anchor" newStyle
-        align = if null alignAttr
-                then "begin"
+        align = if T.null alignAttr
+                then T.pack "begin"
```

Comment:
Don't call `T.pack` on string literals. Instead, we use the GHC extension `OverloadedStrings` (this automatically converts a string literal like `"begin"` into the correct type. This is already present at the top of this file, so it should just work.